fantasybot/telegram/bot.py

```
@@ -0,0 +1,1020 @@
+                    "timeout": 25,
+                })
+                if updates:
+                    for upd in updates:
+                        upd_id = upd.get("update_id", 0)
+                        if upd_id >= self.offset:
+                            self.offset = upd_id + 1
+                        self.handle_update(upd)
+            except KeyboardInterrupt:
+                print("\n[Telegram Bot] Stopping polling daemon...")
+                break
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)